ui/web_remote/main.js

```diff
@@ -55,7 +55,7 @@ function hide_btn_loading(id){
     $('#' + id).html(display);
 }
 
-function add_button(config, click_func){
+function add_button(config, click_func, dest='#button_list'){
```

Comment:
Yeah... it's recent-ish. But well supported. I didn't end up using this now, but I want to in the future so I left it.